components/dashboard/package.json

```
@@ -49,9 +49,11 @@
   "scripts": {
     "start": "craco start",
     "build": "craco build --verbose",
-    "test": "craco test",
+    "test": "yarn test:unit && ./scripts/run-integration-tests.sh",
```

Thanks for adding this helpful script! 🙏

**Suggestion**: I feel like it would be cool to add it as a `pre` task, so that `yarn test:integration:run` can also work out of the box. What do you think of something like this?

```suggestion
    "test": "yarn test:unit && yarn test:integration:run",
    "pretest:integration:run": "./scripts/run-integration-tests.sh",
```

Note: This implies removing the `yarn test:integration:run` from the bottom of the script.

---

I choose the current hierarchy so we can run `yarn test:integration:run` separately, against a dev server in watch mode. The script is just a crutch for the parallelims (dev server x cypress). Let's try this a bit, and adjust when it confuses us.

---

Makes sense to me, thanks! 🚀